Tasks/AzureFunction/task.json

```diff
@@ -61,12 +61,12 @@
 			"name": "headers",
 			"type": "multiLine",
 			"label": "Headers",
-			"required": "false",
-			"defaultValue": "{\n\"Content-Type\":\"application/json\"\n}",
+			"required": "true",
```

Comment:
It blocks, if customer doesn't want anything in headers. Customer has to give some dummy value even he doesn't eant anything. Keep it false only.

Reply:
++

Reply:
fixed.